Source/Scene/QuadtreePrimitive.js

```diff
@@ -667,7 +669,7 @@ define([
     }
 
     function screenSpaceError(primitive, frameState, tile) {
-        if (frameState.mode === SceneMode.SCENE2D) {
+        if (frameState.mode === SceneMode.SCENE2D || frameState.camera.frustum instanceof OrthographicFrustum) {
```

Comment:
`OrthographicOffCenterFrustum` is used in 2D and picking. In 2D, the camera only moves in the yz-plane. Zooming is achieved by adjusting the frustum planes.  For picking, we create a frustum that only contains the 3x3 pixel area around the picked pixel.

`OrthographicFrustum` is centered. The left plane is always the negative of the right plane. This is used in 3D and Columbus view. Because the camera can move freely, we adjust the frustum width based on the distance to the earth, but it is always centered on the camera view direction.

Shadow maps used `OrthographicOffCenterFrustum` but I believe it could use `OrthographicFrustum`.

`PerspectiveOffCenterFrustum` is only used for picking a 3x3 area like picking with an orthographic projection. It could also be used by VR.